src/utils/Api/api_call.js

```diff
@@ -0,0 +1,23 @@
+import axios from "axios";
+import { COINBASEAPI } from "../constants";
+let USDC_PRICE_OF_ETH = 0;
+
+const getDipValue = async (userAsset) => {
+  
+  USDC_PRICE_OF_ETH  =await CoinBaseAPI();
+  console.log(USDC_PRICE_OF_ETH*0.85)
+
+  return (USDC_PRICE_OF_ETH*0.85)
+};
+const CoinBaseAPI = async (userAsset) => {
```

Comment:
why is the nomenclature for const variable names different everywhere?
Let's keep it consistent.
change CoinBaseAPI -> coinBaseAPI ?